src/io/flutter/jxbrowser/JxBrowserManager.java

```diff
@@ -39,12 +41,12 @@
 // the class path.
 public class JxBrowserManager {
   private static JxBrowserManager manager;
-  private static final String DOWNLOAD_PATH = PathManager.getPluginsPath() + File.separatorChar + "flutter-intellij" + File.separatorChar + "jxbrowser";
+  protected static final String DOWNLOAD_PATH = PathManager.getPluginsPath() + File.separatorChar + "flutter-intellij" + File.separatorChar + "jxbrowser";
```

Comment:
feel free to add a @VisibleForTesting annotation to these protected fields to indicate they are only there for testing.